lib/autoyast.rb

```diff
@@ -267,20 +267,26 @@ def apply_changed_files(scope)
 
     @system_description[scope].files.each do |file|
       if file.deleted?
-        @chroot_scripts << "rm -rf '#{file.name}'"
+        @chroot_scripts << "rm -rf '#{quoted(file.name)}'"
       elsif file.directory?
-        @chroot_scripts << "chmod #{file.mode} '#{File.join("/mnt", file.name)}'"
-        @chroot_scripts << "chown #{file.user}:#{file.group} '#{File.join("/mnt", file.name)}'"
+        @chroot_scripts << <<EOF.strip
+chmod #{file.mode} '#{File.join("/mnt", quoted(file.name))}'
+chown #{file.user}:#{file.group} '#{File.join("/mnt", quoted(file.name))}'
+EOF
```

Comment:
I think an EOF is not as readable but it is not that important especially if the line length restriction was the problem.